ng-client/src/app/project/project-list/project-list.component.html

```diff
@@ -8,10 +8,14 @@
           <md-icon class="md-24">add</md-icon>
         </button>
       </div>
-    </md-list-item>
-    <md-list-item *ngFor="let project of projects">
-      <h3 md-line>{{project.name}}</h3>
-      <p md-line class="demo-secondary-text">{{project.description}}</p>
-    </md-list-item>
-  </md-list>
+      <md-list-item *ngIf="projects.length === 0">
+        <p md-line class="empty-list-message">You don't have any projects yet!</p>
+      </md-list-item>
+      <md-list-item *ngFor="let project of projects">
+        <md-icon md-list-icon>business</md-icon>
+        <h4 md-line class="project-list-item-name">{{project.name}}</h4>
+        <p md-line class="project-list-item-description">{{project.description}}</p>
+      </md-list-item>
+    </md-list>
```

Comment:
Not sure if the md-list-item above needs to be closing, or if it needs to be removed, but tests are failing because md-list isn't expected to close yet